docs/source/notebook.rst

```diff
@@ -420,9 +427,8 @@ The Jupyter Notebook is officially supported the latest stable version the follo
 * Safari
 * Firefox
 
-The is mainly due to the notebook's usage of WebSockets and the flexible box model.
-
-The following browsers are unsupported:
+Principally due to the notebook's usage of WebSockets and the flexible box model,
```

Comment:
These older browser versions are unsupported due to the notebook's usage of WebSockets and more current web browser features: